monkey/common/agent_plugins/agent_plugin_repository_index.py

```diff
@@ -26,6 +22,8 @@ class AgentPluginRepositoryIndex(MutableInfectionMonkeyBaseModel):
         :param plugins: Plugins' metadata, segregated by type and sorted by version
     """
 
+    model_config = ConfigDict(arbitrary_types_allowed=True, use_enum_values=True)
```

Comment:
Shouldn't this be inheriting from `MutableInfectionMonkeyModelConfig`?

Reply:
Nope, UTs are in [monkeytypes](https://github.com/guardicode/monkeytypes/blob/dev/tests/test_base_models.py) about this